### PR TITLE
fix: adjust the margins and paddings of the entire mockup

### PR DESCRIPTION
**Pull request by GusBaamonde**
- Add the keyframe function that has styled component to make animations
- Add progressbar animation for color bars
- Add a variable file for spacing normalization
- Add a varialbes file for normalization of font sizes
- Design validated by @santiagopaolucci 

reportsTrafficSources

### CODE REVIEW

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.test.js [hunk 1]
```diff
@@ -70,6 +70,6 @@ describe('reports traffic sources', () => {
     expect(getByText('trafficSources.email'));
     expect(getByText('trafficSources.social'));
     // TODO: for now only can check english language local because for local has small-icu and need full-icu
-    expect(getByText('66.67%'));
+    // expect(getByText('66.67%'));
```
Comment (cbernat) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.test.js]: I think you should find the way to fix this test, or delete this line.

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js [hunk 2]
```diff
@@ -55,14 +55,15 @@ const ReportsTrafficSources = function({ domainName, dateFrom, dependencies: { d
                     />
                   </h6>
                   <span>
-                    {trafficSource.quantity} (
+                    {trafficSource.quantity}
                     <span>
+                      (
```
Comment (andresmoschini) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js]: Just curiosity, why did you move the parenthesis inside the span?

File: src/components/styles/spacings.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export default {
```
Comment (andresmoschini) [inline src/components/styles/spacings.js]: In general, is not a good idea exporting default, what do you think about unifying all these things in an only file and export _spacings_, _fonts_, etc?

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js [hunk 4]
```diff
@@ -1,5 +1,17 @@
-import styled, { css } from 'styled-components';
+import styled, { css, keyframes } from 'styled-components';
 import colors from '../../styles/colors';
+import fonts from '../../styles/fonts';
+import spacings from '../../styles/spacings';
+
+const progressBar = keyframes`
```
Comment (andresmoschini) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js]: Wha is `keyframes`? do you have a link with a simple introduction?

Comment (GusBaamonde) [reply src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js]: Keyframes is to make animations
https://www.styled-components.com/docs/basics#animations

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js (hunk 2 above)
Comment (GusBaamonde) [reply src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js]: the design required a space between `span` and `span`. By having the `(` outside the span it was separated into three parts, and it looked bad

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.test.js (hunk 1 above)
Comment (GusBaamonde) [reply src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.test.js]: ok

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js (hunk 4 above)
Comment (andresmoschini) [reply src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js]: Nice! thanks!

**Comment by santiagopaolucci** [comment]
I like very much!

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+  0% {
+    transform: scaleX(0);
+  }
+
+  100% {
+    transform: scaleX(1);
+  }
+`;
```
Comment (fcoronelmakingsense) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js]: doesn´t work use only `to {
    transform: scaleX(1);
  }` ?

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js [hunk 6]
```diff
@@ -8,7 +18,7 @@ export const ListContainer = styled.div`
     background: ${colors.smoothGrey};
   }
   & > div:nth-child(n + 4) {
-    margin-top: 10px;
+    margin-top: ${spacings.spacesLv2};
```
Comment (fcoronelmakingsense) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js]: it's ok the name `Lv` or it's better use `Lvl` ?

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js [hunk 7]
```diff
@@ -17,50 +27,77 @@ export const ListItem = styled.div`
...
+    top: 30px;
+    left: 0;
+    border: 1px solid ${colors.faintGrey};
+  }
+
+  span {
+    font-size: ${fonts.fontLv3};
+  }
+
+  span span {
```
Comment (fcoronelmakingsense) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js]:
```suggestion
  span {
   font-size: ${fonts.fontLv3};
   span { 
      margin-left: ${spacings.spacesLv1};
   }
```

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js [hunk 8]
```diff
@@ -17,50 +27,77 @@ export const ListItem = styled.div`
...
     display: flex;
     justify-content: space-between;
     align-items: flex-end;
-    margin-top: 15px;
+    margin-top: ${spacings.spacesLv5};
     & > div {
       width: 70%;
     }
+
+    span span {
```
Comment (fcoronelmakingsense) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js]: same of last comment

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.test.js [hunk 9]
```diff
@@ -69,7 +69,5 @@ describe('reports traffic sources', () => {
     await waitForDomChange();
     expect(getByText('trafficSources.email'));
     expect(getByText('trafficSources.social'));
-    // TODO: for now only can check english language local because for local has small-icu and need full-icu
-    expect(getByText('66.67%'));
```
Comment (fcoronelmakingsense) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.test.js]: why removed?

File: src/components/styles/styles.js [hunk 10]
```diff
@@ -0,0 +1,45 @@
+/* Spacings for margenes and paddings */
+export const spacings = {
```
Comment (fcoronelmakingsense) [inline src/components/styles/styles.js]: I read it's recommended use multiple files, like `colors, spacings, fonts` because in the future can growing up and it's more difficult to maintain and less clear all in the same file

Comment (cbernat) [reply src/components/styles/styles.js]: I agree with @fcoronelmakingsense better keep them separated.

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.test.js (hunk 9 above)
Comment (cbernat) [reply src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.test.js]: There were two options, either remove the line or fix the test. With the animation perhaps this is not always like it's expected.

Comment (GusBaamonde) [reply src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.test.js]: I leave it commented?

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js (hunk 5 above)
Comment (GusBaamonde) [reply src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js]: It doesn't work like that, you have to indicate the start

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js (hunk 6 above)
Comment (GusBaamonde) [reply src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js]: I change it to spacesLvl
